packages/remix-dev/config/defaults/node/entry.server.react-stream.tsx

```diff
@@ -103,8 +112,13 @@ function handleBrowserRequest(
           reject(error);
         },
         onError(error: unknown) {
-          console.error(error);
           responseStatusCode = 500;
+          // Log streaming rendering errors from inside the shell.  Don't log
+          // errors encountered during initial shell rendering since they'll
+          // reject and get logged in handleDocumentRequest.
+          if (shellRendered) {
+            console.error(error);
+          }
```

Comment:
Add this to the templates as well

Reply:
Interesting... do you typically review/comment on your own PRs?

Reply:
I guess it works like a TODO list... I may have to start doing that.

Reply:
A self review also helps when other people review it 🎉

Reply:
haha all of the above!  Usually the first set of comments is me providing context for the reviewer.  That also sometimes turns up things I forgot about and will leave myself comments as TODOs if I don't have the time to fix them right then.  And then this specific set of comments are from some pair-reviewing Jacob and I have been doing lately where we review PRs on a screenshare.

I'm very list-driven (as evidence dby the post it notes all over my desk).  I also often do actual TODO lists for myself in the PR description: 

<img width="143" alt="Screenshot 2023-06-01 at 4 40 46 PM" src="https://github.com/remix-run/remix/assets/1609022/11a37568-7158-40ff-9cbe-d026a19055f8">